src/ImageAPI/Processing/Drawing/DrawLeftAmount.cs

```diff
@@ -1,20 +1,14 @@
-﻿using ImageAPI.Utils;
-using SixLabors.Fonts;
-using SixLabors.ImageSharp;
+﻿using SixLabors.ImageSharp;
 
 namespace ImageAPI.Processing.Drawing;
 
 public class DrawLeftAmount : ToDrawing
 {
-    public override string Text { get; }
-    public override Font Font { get; }
-    public override PointF Coordinates { get; }
-
     public DrawLeftAmount(object leftAmount)
-    {
-        Text = leftAmount.ToString()!;
-        // TODO: Create class which provide caching fonts by fontSize. This class get exist font or create new and save it.
-        Font = new ResourcesLoader().LoadFontFromEmbeddedResources(AmountFontSize);
-        Coordinates = new PointF(ImageSize.Width - 400, ImageSize.Height - 290);
-    }
+        : base(
+            text: leftAmount.ToString()!,
```

Comment:
"Left Amount"

Reply:
https://github.com/The-Poolz/LockDeal-NFT-API/issues/182